### ISSUE TITLE
Enhancement: add label to mouse-over tooltip

### ISSUE THREAD

**Issue by dhowe** [jf/fosx-label#1]
Excellent addon. Not sure how much work this would require, but would be even more excellent if the label could be displayed on mouse-over, something like "Firefox (Label)", rather than just "Firefox":

![screen shot 2015-11-30 at 5 57 25 pm](https://cloud.githubusercontent.com/assets/737638/11468605/d9461e76-978b-11e5-8b82-058a981800cb.png)

**Comment by jf** [comment]
Good idea; I'm not so sure how much functionality is exposed to the plugin system right now, though, to be able to do this. Needs some research.... Have u seen any other plugin (so I can reference code) that's been able to do this?
